aries-site/src/examples/templates/filtering/FilteringCards.js

```diff
@@ -438,6 +459,44 @@ StatusFilter.propTypes = {
   setStatus: PropTypes.func.isRequired,
 };
 
+const NameFilter = ({ filters, setFilters, setName, name }) => (
+  <FormField label="User" name="user-filter-a" htmlFor="user-filter-a">
+    <Box
+      width="medium"
+      overflow="scroll"
```

Comment:
@taysea Can we chat through the desired experience here? The many scroll bars is a bit difficult to interact with. When I look at the Figma file, I thought it is advocating for a "show more" anchor: https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=3%3A1595

Am I interpreting the Figma files incorrectly.

Reply:
Yes, we can talk through it. Greg wanted to demonstrate both an anchor with "show more" and a scrolling CheckBoxGroup (he's adding it to the Figma file), but I do see that the multiple scroll bars can be tough. Maybe a scrolling checkboxgroup would be better used in cases where a layer is not scrollable.

Reply:
I will also be demonstrating the "show more" anchor in an example

Reply:
I've updated so that the scrolling side layer utilizes the "Show more" anchor and the center layer example demonstrates a scrolling checkboxgroup.